src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,21 @@
-export const UserInfo = () => {};
+import React from 'react';
+import './UserInfo.scss';
+import { User } from '../../types/User';
+
+interface Props {
+  user: User,
+  // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
Avoid disabling eslint :)

Reply:
ok
task sometimes seems to be more complex than it actually is))